.circleci/config.yml

```diff
@@ -63,7 +63,7 @@ jobs:
     executor: docker-with-node
     steps:
       - with-workspace
-      - run: yarn test:coverage
+      - run: yarn test
```

Comment:
I thought that the next step (on line 67) sends the coverage somewhere. Is it not the case?

Reply:
I removed it as it will run `test:coverage` for each package, but now added it back.